sae_lens/training/sparse_autoencoder.py

```diff
@@ -180,26 +196,28 @@ def initialize_b_dec_with_mean(self, all_activations: torch.Tensor):
 
     @torch.no_grad()
     def set_decoder_norm_to_unit_norm(self):
-        self.W_dec.data /= torch.norm(self.W_dec.data, dim=1, keepdim=True)
```

Comment:
I think the if statement should go outside of this function (would be much more intuitive)